bindings/nodejs/lib/utils/Utils.ts

```diff
@@ -28,7 +28,7 @@ export class Utils {
     /**
      * Returns a hex encoded seed for a mnemonic.
      */
-    static mnemonicToHexSeed(mnemonic: string): string {
+    static mnemonicToHexSeed(mnemonic: string): HexEncodedString  {
```

Comment:
You added one space too many :trollface:

Reply:
stupid formatter 🗡️